### ISSUE TITLE
error: ‘pid_t gettid()’ was declared ‘extern’ and later ‘static’ 

### ISSUE THREAD

**Issue by gi9obru0etuPlc6wozls** [treefrogframework/treefrog-framework#274]
I'm getting the following errors when making src:
```
    g++ -c -pipe -g -std=gnu++1y -Wall -W -D_REENTRANT -fPIC -DTF_MAKEDLL -DQT_SQL_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_XML_LIB -DQT_CORE_LIB -I. -I../include -I../3rdparty/lz4/lib -I../3rdparty/mongo-driver/src/mongoc -I../3rdparty/mongo-driver/src/libbson/src/bson -isystem /usr/include/qt -isystem /usr/include/qt/QtSql -isystem /usr/include/qt/QtQml -isystem /usr/include/qt/QtNetwork -isystem /usr/include/qt/QtXml -isystem /usr/include/qt/QtCore -I.obj -I/usr/lib/qt/mkspecs/linux-g++ -o .obj/tapplicationserverbase.o tapplicationserverbase.cpp
    In file included from tapplicationserverbase.h:8,
                     from tapplicationserverbase.cpp:12:
    ./tfcore_unix.h:26:21: error: ‘pid_t gettid()’ was declared ‘extern’ and later ‘static’ [-fpermissive]
       26 | static inline pid_t gettid()
          |                     ^~~~~~
    In file included from /usr/include/unistd.h:1170,
                     from /usr/lib/qt/mkspecs/linux-g++/qplatformdefs.h:56,
                     from ../include/../src/twebapplication.h:11,
                     from ../include/twebapplication.h:1,
                     from ../include/TWebApplication:1,
                     from tapplicationserverbase.cpp:8:
    /usr/include/bits/unistd_ext.h:34:16: note: previous declaration of ‘__pid_t gettid()’
       34 | extern __pid_t gettid (void) __THROW;
          |                ^~~~~~
    make: *** [Makefile:2495: .obj/tapplicationserverbase.o] Error 1
    [xyzzy@aristotle src]$ gcc --version
    gcc (GCC) 9.2.0
    Copyright (C) 2019 Free Software Foundation, Inc.
    This is free software; see the source for copying conditions.  There is NO
    warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

```

**Comment by treefrogframework** [comment]
Fixed in 44a111f86049d86fbcdd5c9092477c17452788c5.
Thanks!

**Comment by gi9obru0etuPlc6wozls** [comment]
Thank you!
